controller.go

```diff
@@ -206,6 +196,66 @@ func New(cfgOptions ...config.Option) (NetworkController, error) {
 	return c, nil
 }
 
+func (c *controller) ReloadConfiguration(cfgOptions ...config.Option) error {
+	// For now we accept the configuration reload only as a mean
+	// to provide a global store configuration after boot
+	c.Lock()
+	for _, ds := range c.stores {
+		if ds.Scope() == datastore.GlobalScope {
+			c.Unlock()
+			return types.ForbiddenErrorf(
```

Comment:
This must fail ONLY if the config has changed.
If the config is exactly the same, this function must return immediately without any error.